OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        his company was sold. A.H. Huschke &amp; Co acquired a high
+                                        ‘reputation for luxury cottons’ and other ‘quality goods for
+                                        the fashion market’</desc>
+                                    <note>(see: <bibl>
+                                            <ptr target="bm:Nenadic2019"/>
+                                        </bibl>).</note>
+                                </item>
+
+                                <item xml:id="e4">
+                                    <locus target="#1v #2v"/>
+                                    <desc> A white sticker, with the name of <persName
+                                            ref="PRS5782JuelJen"/> (<foreign xml:lang="gez">ቤንት፡
+                                            ዩውል፡ የንሰን።</foreign>) printed on it, and the signatures
+                                        ‘Ms. Aeth. f. 29’ written in pencil, is respectively glued
+                                        on <locus target="#1v"/> and on <locus target="#2v"/> .
+                                        Other two stickers with the signature ‘Ms 19’ printed on
+                                        them are glued on <locus target="#1v"/>.</desc>
+                                </item>
+                            </list>
+                        </additions>
+
+                        <bindingDesc>
+                            <binding contemporary="false" xml:id="binding">
+                                <decoNote xml:id="b1">Two<material key="wood">wooden</material>
+                                    boards covered with tooled reddish brown tooled <material
+                                        key="leather"/>. A niche for a mirror is carved into the
+                                    inner side of the front board (the mirror is lost, but traces of
+                                    adhesive are still visible). The inner side of the back board is
+                                    decorated with a silk inlay (coloured gold, blue, and
+                                    red).</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">The textile inlays are
+                                    either cotton or silk (see <ref target="#e3"/>). <material
+                                        key="wood"/>
+                                    <material key="leather"/>
+                                    <material key="textile"/>
+                                </decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="reddish brown"
+                                    >Blind-tooling decoration on the full leather cover including
+                                    turn-ins, edges, and spine. The decoration was carefully applied
+                                    using different tools.</decoNote>
```

Comment:
I am fine with this level of description, but I think we need to be consistent, either we describe the tools, like Massimo did, or we do not; I don't think it's necessary to describe the tools in great details, but we can talk about this.